unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -36,6 +36,7 @@ private void Awake()
         {
             model = new AvatarModel();
             currentPlayerInfoCardId = Resources.Load<StringVariable>(CURRENT_PLAYER_ID);
+            avatarRenderer.OnImpostorAlphaValueUpdate += (newAlphaValue) => avatarMovementController.movementLerpWait = newAlphaValue > 0.1f ? DataStore.i.avatarsLOD.impostorSettings.movementInterpolationWait.Get() : 0f;
```

Comment:
We should use a method (or a delegate) and unsubscribe it when disposing `AvatarShape`.

Reply:
Done